source/type/type.cpp

```diff
@@ -423,7 +429,10 @@ VJ* MetaVehicleJourney::impl_create_vj(const std::string& uri,
     vj_ptr->realtime_level = level;
     vj_ptr->shift = 0;
     if (!sts.empty()) {
-        vj_ptr->shift = sts.front().arrival_time / (ndtu::SECONDS_PER_DAY);
+        const auto first_st_it = std::min_element(sts.begin(), sts.end(), [](const StopTime& st1, const StopTime& st2) {
```

Comment:
earliest_stop_time is on a vj, not a vector of stop_times. I should probably move the min_element in a separate function taking a vector and just call this one in types::VehicleJourney::earliest_stop_time, type::VehicleJourney::earliest_stop_time and in impl_create_vj. Should I add it in type_utils.h maybe?